pkg/okta/okta.go

```diff
@@ -109,12 +120,18 @@ func newJwtVerifier(clientID string, issuer string) *jwtverifier.JwtVerifier {
 type oktaMiddlewareFactory struct {
 	handlers.HandlerBase
 	verifier *jwtverifier.JwtVerifier
+	codeGRT  string
 }
 
 // NewOktaAuthorizeMiddleware returns a wrapper for HandlerFunc to authorize with Okta
-func NewOktaAuthorizeMiddleware(base handlers.HandlerBase, clientID string, issuer string) func(http.Handler) http.Handler {
+func NewOktaAuthorizeMiddleware(base handlers.HandlerBase, clientID string, issuer string, ovrdGRT bool) func(http.Handler) http.Handler {
```

Comment:
I think we should spell out overrideGRT or write it in the boolean pattern of "is________" or similar.

Reply:
Yup, spelled out